postBuild

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
We shouldn't need a `postBuild` file, as all this happens in the Dockerfile. Even for things like binder, you can create the Dockerfile and let binder serve the image itself.